docs/adguard-for-windows/overview.md

```diff
@@ -156,12 +159,33 @@ By clicking the Support tab you will open a dialog box through which you can rep
 
 [License]()
 
-In this tab, everything is very simple: three buttons and a minimum amount of information. Here you can:
-* find out the license type (natli build, veta or release), the number of devices available for simultaneous connection to AdGuard, the license expiration date;
+This tab contains information about your license. Here you can:
+* find out the license type (nightly build, beta or release), the number of devices available for simultaneous connection to AdGuard, the license expiration date;
 * click the button to go to the license purchase page, and activate, reset or recover a license. 
 
 ### About 
 
 [About]()
 
-In addition to information about the current version of the program, this tab provides some useful links to the Licese Agreement, Privacy Policy, Acknowledgements, Discussions, and Version history pages. 
\ No newline at end of file
+In addition to information about the current version of the program, this tab provides some useful links to the Licese Agreement, Privacy Policy, Acknowledgements, Discussions, and Version history pages. 
+
+### Check for updates
+
+To check for new AdGuard versions, click the rotating arrows icon in the upper right corner of the app's main screen.
+
+### Tray menu
+
+For quick control of the program through the tray menu, all the basic frequently used settings are available. To call the menu, right-click the AdGuard icon in the tray and: 
```

Comment:
>For quick control of the program through the tray menu, all the basic frequently used settings are available.

Лучше так:

>To save you time navigating around AdGuard menus, many of the frequently used options and settings are available from the tray menu. To call the menu, right-click the AdGuard icon in the tray. It will offer you the following options: